src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -77,6 +78,9 @@ private static void setupDrivers(Application application) {
         DriverFeature.getDriverManager().setCurrentDriver(basicLineDriver);
         driverComposite.addDriver(basicLineDriver);  // addidtion to composite
 
+        Job2dDriver macroDriver = new RecordMacroDriver(basicLineDriver);
+        DriverFeature.addDriver("Macro driver", macroDriver);
+
```

Comment:
OK, but macro should be applied automatically. Use the solution from PR #14 (e.g., see ApplyDriverDecoratorsSubscriber 
 class)